LocalSettings.php

```diff
@@ -32,7 +32,7 @@ function loadenv($envName, $default = "") {
 # $wgDisableOutputCompression = true;
 
 $wgSitename = loadenv('MEDIAWIKI_SITE_NAME', "Tamil Wiki");
-$wgMetaNamespace = loadenv('MEDIAWIKI_META_NAMESPACE', "Literary_Tamil_Wiki");
+$wgMetaNamespace = loadenv('MEDIAWIKI_META_NAMESPACE', "Tamil_Wiki");
```

Comment:
What is the need for changing this now?

Reply:
Meta Pages.. pages about the website, guidelines now reside in the project name space.. 
eg: https://tamil.wiki/wiki/Literary_Tamil_Wiki:அறிவுறுத்தல்கள்

While going public, would be better to have this as tamil wiki. We will be adding more pages in this name space, including details of contributors and other teams etc. 

Vijay mentioned that it can be changed in environment - within server. You can decide the right place / way to make the change.

Reply:
Also, we have to add 
$wgRestrictDisplayTitle = false; to the local setting.php.. 

this will allow us to change the title displayed in a wiki page.. Title for the english home page is Main_Page_(English), we can change it to Tamil Wiki..

Reply:
hmm.. I dont know the impact. I will let others decide. @arulrajnet @vijayabharathib ?